shim/stub.go

```diff
@@ -132,6 +133,18 @@ func (s *ChaincodeStub) GetDecorations() map[string][]byte {
 	return s.decorations
 }
 
+// GetMSPID returns the local mspid of the peer by checking the CORE_PEER_LOCALMSPID
+// env var and returns an error if the env var is not set
```

Comment:
nit: doc should be written in complete statements and end with a period. Since I know this doc isn't great, this is business as usual.